Public/Src/Cache/ContentStore/Distributed/NuCache/InMemory/MemoryContentLocationEventStore.cs

```diff
@@ -138,6 +138,16 @@ public void Send(ContentLocationEventData eventData)
                     OnEvent(eventData);
                 }
             }
+
+            /// <summary>
+            /// The above function, without the lock. Used because of perf benchmarks getting contention on

```

Comment:
It is better to clearly indicate which function, instead of saying "the above function".